internal/state/conditions/conditions.go

```diff
@@ -83,6 +88,16 @@ func NewRouteAccepted() Condition {
 	}
 }
 
+// NewRouteUnsupportedValue returns a Condition that indicates that the HTTPRoute includes an unsupported value.
+func NewRouteUnsupportedValue(msg string) Condition {
+	return Condition{
+		Type:    string(v1beta1.RouteConditionAccepted),
+		Status:  metav1.ConditionFalse,
```

Comment:
yep, it is always false.  partial validity is not specified right now -- https://github.com/kubernetes-sigs/gateway-api/issues/1696 

However, defining Accepted False for the case of both valid and invalid routing rules present doesn't make sense - because the route will be handled by NKG.

What do you think about adding a `PartiallyAccepted` condition for now? It will include errors for invalid rules for a partially valid route. If all rules are invalid, we will use `Accepted` condition with status False as we do now

this will affect this code in `httproute.go`
```
	if len(allRulesErrs) > 0 {
		// FIXME(pleshakov): We report Accepted condition with status False even if some of the rules are valid.
		// The spec is not clear about such cases. However, this issue
		// https://github.com/kubernetes-sigs/gateway-api/issues/1696 might clarify the situation.
		msg := allRulesErrs.ToAggregate().Error()
		if !atLeastOneValid {
			msg = "All rules are invalid: " + msg
		}
		r.Conditions = append(r.Conditions, conditions.NewRouteUnsupportedValue(msg))
	}
```

Reply:
I think a `PartiallyAccepted` condition makes sense. Will adding that condition hurt us with the conformance tests?

Reply:
discussed offline: will use TODO condition

Reply:
https://github.com/nginxinc/nginx-kubernetes-gateway/pull/455/commits/79c2c8146c42be79bfbd217dbe1ac332e9178a40